src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlSearchOptions.cs

```diff
@@ -39,5 +44,26 @@ public SqlSearchOptions(SearchOptions searchOptions)
         /// Performs a shallow clone of this instance
         /// </summary>
         public SqlSearchOptions CloneSqlSearchOptions() => (SqlSearchOptions)MemberwiseClone();
+
+        public string GetHash()
+        {
+            // I don't think UnsupportedSearchParams is needed as it isn't used in the actual search
```

Comment:
Is this comment meant to be committed? 🍒

Reply:
I should reword it, but I think the point is helpful to understand why I didn't include that value in the hash.